src/models/UserModel.js

```diff
@@ -0,0 +1,32 @@
+import mongoose from 'mongoose'
+import validator from 'validator'
+
+
+const {Schema, model} = mongoose
+const { isEmail } = validator 
+
+const userSchema = new Schema (
+    {
+        firstName: {
+            type: String,
+            required: true
+        },
+        lastName: {
+            type: String,
+            required: true
+        },
+        email: {
```

Comment:
Codacy has a fix for the issue: [Delete `····`](https://app.codacy.com/gh/Denniman/group-four-BE/pullRequest?prid=7801432)
```suggestion
    email: {
```